test/instrumentations/express/app/express_app.js

```diff
@@ -47,7 +47,7 @@ app.get('/basic', async (_, res) => {
 
 server = app.listen(0, () => {
   const port = server.address().port;
-  console.info('Listening on port ' + port);
+  console.error('Listening on port ' + port);
```

Comment:
hmmm... i was under the impression that if you console.log, then the logs will always be displayed but on stdout, which seems like the best thing to do in this case

Reply:
In the test app we explicitly listen to `stderr`